BT enable lightbar+rumble

firodj opened the pull request:
here is when I try to re-check and change to output reports, to enable rumble + lightbar over BT.

Ryochan7 commented:
Has this been investigated any further? Has output report 0x11 been tested again?

firodj commented:
for 0x11 also affected to use 0xf3 instead 0xf7, but I've added a flag which could be enabled/disable when using copy-cat DS4. Hence this PR is obsolete will closed. Moved into: https://github.com/Ryochan7/DS4Windows/pull/1839
